database/seeders/ProjectSeeder.php

```
@@ -9,10 +9,14 @@ class ProjectSeeder extends Seeder
 {
     public function run(): void
     {
-        Project::create([
+        $project = Project::create([
+            'uuid' => 'project',
             'name' => 'My first project',
             'description' => 'This is a test project in development',
             'team_id' => 0,
         ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Project' in method 'run'. (undefined)

(StaticAccess)